src/object.d

```diff
@@ -2094,7 +2094,7 @@ auto byKeyValue(T : V[K], K, V)(T aa) pure nothrow @nogc
         AARange r;
 
     pure nothrow @nogc:
-        @property bool empty() { return _aaRangeEmpty(r); }
+        @property bool empty() @safe { return () @trusted { return _aaRangeEmpty(r); } (); }
```

Comment:
For DMD there is an overhead in these lamdas and in this case there is no need for the lamda if you replace `@safe` with `@trusted`

Reply:
I thought Kenji fixed the compiler so when the compiler detects this pattern it always inlines the lambda.

Reply:
This should simply be marked `@safe`, and the implementation of `_aaRangeEmpty` marked `@safe`.  We should avoid `@trusted` wherever we can.